.circleci/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 
       - run:
           name: configure
-          command: ./configure --config dev --host dev.koding.com:8090 --hostname dev.koding.com --countlyApiPort "80"
+          command: ./configure --config dev --version $(git rev-parse --short HEAD) --host dev.koding.com:8090 --hostname dev.koding.com --countlyApiPort "80"
```

Comment:
[require('git-rev-sync').short](https://github.com/kurttheviking/git-rev-sync-js/blob/master/index.js#L117-L119) just gets the first 8 characters.

[git rev-parse --short](https://github.com/git/git/blob/master/builtin/rev-parse.c#L161) [finds the unique abbreviation](https://github.com/git/git/blob/master/sha1_name.c#L466-L509) of the revision. I think the former is okay for development purposes only.